docs/products/opensearch/howto/os-version-upgrade.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Upgrade Aiven for OpenSearch®
+sidebar_label: Upgrade version
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Aiven for OpenSearch® allows you to choose the version that best fits your needs and upgrade when ready.
+
+## Multi-version: how it works
+
+Aiven for OpenSearch supports multiple minor and major versions.
+
+### Default version
+
+When
+[creating a Aiven for OpenSearch service](/docs/products/opensearch/get-started#create-an-aiven-for-opensearch-service),
+you can select the starting version. Otherwise, the service gets the latest version
```

Comment:
On release we're not going to change default version, so for the first time it will be still 2.17, not latest